js/src/setup-mc/setup-stepper/stepNameKeyMap.js

```diff
@@ -1,7 +1,9 @@
 const stepNameKeyMap = {
-	accounts: 1,
-	target_audience: 2,
-	shipping_and_taxes: 3,
+	accounts: '1',
+	target_audience: '2',
+	shipping_and_taxes: '3',
```

Comment:
The type should be string: https://github.com/woocommerce/woocommerce-admin/blob/v2.1.5/packages/components/src/stepper/index.js#L128-L131

Reply:
:nail_care: :scroll:  I wonder if they are string keys, and we provide a sorted `steps` array, maybe we don't need a human-readable-key-string-> number-kay-string map, and we can live with a set of direct `accounts, target_audience, shipping_and_taxes, store_requirements` keys?

Reply:
The number values in string types are used to determine if it can go to a certain step when clicking on **Stepper**:

https://github.com/woocommerce/google-listings-and-ads/blob/3adc30a8fc5bdfcf08cc97160b91750818c1c64f/js/src/setup-mc/setup-stepper/saved-setup-stepper.js#L41-L45

Even if we could change it as below, I think it's good to keep it as is. 

```js
const stepNameKeys = [
	'accounts',
	'target_audience',
	'shipping_and_taxes',
	'store_requirements',
];

const handleStepClick = ( stepKey ) => {
	if ( stepNameKeys.indexOf( stepKey ) <= stepNameKeys.indexOf( savedStep ) ) {
		setStep( stepKey );
	}
};
```